.github/workflows/ci.yml

```diff
@@ -0,0 +1,47 @@
+name: iroha-java CI
+
+on:
+  push:
+
+env:
+  BUILD_DIR: client/build
+  SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+  CACHE_VERSION: v01
+  CACHE_PATHS: |
+    ~/.gradle/caches
+    ~/.gradle/wrapper
+jobs:
+  Linux:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-16.04
```

Comment:
Why do you need such old ubuntu? 16.04 has end of life on April 2021. You will have a problem with JDK security updates!